src/db/transaction_repository.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+class TransactionRepository {
+    private $db;
+    private $transactions_table;
+
+    public function __construct($wpdb) {
+        $this->db = $wpdb;
+        $this->transactions_table = $this->db->prefix . '_vpos_woocommerce_transacations_' . VPOS_VERSION;
+    }
+    
+    public function get_transaction($uuid) {  
+        return $this->db->get_results(
+          "SELECT * FROM $this->transactions_table WHERE ID = '$uuid'"
+        );
+    }
+
+    public function insert_transaction($tranasction) {
+        $this->db->insert( 
+          $this->transactions_table, 
+          array(
+            'id' => $tranasction->get_uuid(),
+            'transaction_id' => $tranasction->get_transaction_id(),
+            'status' => $tranasction->get_status(),
+            'type' => $tranasction->get_type(),
+            'amount' => $tranasction->get_amount(),
+            'mobile' => $tranasction->get_mobile(), 
+            'status_reason' => $tranasction->get_status_reason(),
+            'created_at' => current_time('mysql'),
+            'updated_at' => null
+          )
+        );
+    }
+
+    // TODO: This should probably be moved to a different class since it is not the 
```

Comment:
I was thinking of putting in like db_init.php script since that function would be used to actually create the table.
The functions that are responsible for interacting with that table would be in the repository. 

But I do think that having it closer to the repository makes it easier to change things since they are all encapsulated in the same class and therefore are closer to each other.